ixp_tracker/stats.py

```diff
@@ -96,6 +100,8 @@ def generate_stats(geo_lookup: ASNGeoLookup, stats_date: datetime = None):
                 "rs_peering_rate": rs_peering_rate,
                 "members_joined_last_12_months": len(members_joined),
                 "members_left_last_12_months": len(members_left),
+                "growth_members": growth_members,
+                "growth_members_percent": (growth_members / members_last_month) if members_last_month > 0 else 1,
```

Comment:
Technically if there were zero members last month it's infinite growth but that's not helpful so set it to 1 (i.e. 100% growth).